### PR TITLE
update postgresql default cpu request

### PR DESCRIPTION
**Pull request by ivukotic**
addressing https://github.com/ssl-hep/ServiceX/issues/466

### CODE REVIEW

**Comment by sthapa** [comment]
This should do the trick based on the bitnami docs but would need testing

File: helm/servicex/values.yaml [hunk 1]
```diff
@@ -88,6 +88,9 @@ postgresql:
         database: servicex
         postgresPassword: leftfoot1
   primary:
+    resources:
+      requests:
+        cpu: 2
```
Comment (BenGalewsky) [inline helm/servicex/values.yaml]: I really like keeping the default `values.yaml` useful for developers or CI jobs. I think. this belongs in the production flux configs and not here.

Comment (ivukotic) [reply helm/servicex/values.yaml]: is cpu:1 OK with you? Or at least 500m ?

Comment (BenGalewsky) [reply helm/servicex/values.yaml]: I don't have any problem with the bitnami default
```
  resources:
    limits: {}
    requests:
      memory: 256Mi
      cpu: 250m
```

That's good for developers and anyone else should really do what makes sense for their production deployment

Comment (ivukotic) [reply helm/servicex/values.yaml]: do you mind if we still spell 250m here, it is not very obvious where this is set, and even that this is an issue ?

**Pull request merged**